src/libstd/io/stdio.rs

```diff
@@ -872,6 +874,82 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Equivalent to the [`input`] method except that a prompt
+/// is printed before capturing input. This function returns
+/// either success ([`Ok`]) or failure ([`Err`]) containing
+/// the input. See the [`io::Result`] module documentation
+/// for details.
```

Comment:
```suggestion
/// Read a [`String`] from [standard input](Stdin). The
/// trailing newline is stripped. Errors on EOF (Ctrl-D).
///
/// Equivalent to [`input`] with an extra step to print a
/// prompt before capturing input.
```
Stolen from python `input`.